corehq/apps/app_manager/views/modules.py

```diff
@@ -1753,6 +1757,16 @@ def get(self, request, domain):
         })
 
 
+class AllCaseTypesView(LoginAndDomainMixin, View):
+    urlname = 'all_case_types'
+
+    def get(self, request, domain):
+        return JsonResponse({
+            'existing_case_types': list(get_case_types_for_domain(domain)),
```

Comment:
this still misses cases that are part of an application but have not yet been submitted

Reply:
3480c0c29216dde3011dfeef6af1ffaa7492810e